heron/explorer/src/python/main.py

```diff
@@ -28,7 +28,7 @@
 import heron.explorer.src.python.topologies as topologies
 import heron.common.src.python.utils.config as config
 import heron.explorer.src.python.version as version
-from heron.common.src.python.color import Log
+import heron.common.src.python.utils.log as log
```

Comment:
Good suggestion but log module has the logger object called `Log`. There will be name clashing.

> On Aug 2, 2016, at 12:05, Karthik Ramasamy <notifications@github.com> wrote:
> 
> In heron/explorer/src/python/main.py:
> 
> > @@ -28,7 +28,7 @@
> >  import heron.explorer.src.python.topologies as topologies
> >  import heron.common.src.python.utils.config as config
> >  import heron.explorer.src.python.version as version
> > -from heron.common.src.python.color import Log
> > +import heron.common.src.python.utils.log as log
> can you import as Log rather than log?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>